src/lib/github/addIssueToRepo.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ *  API Function that adds an issue with comments to a repository
+ */
+
+ export interface IssuesWithComments {
+    title: string;
+    comments: string[];
+ }
+
+ export default async function addIssueToRepo(context: any, issuesWithComments: IssuesWithComments[] ): Promise<void> {
```

Comment:
This function should be better named to something like; `addIssuesToRepo`.

And could you please explain in a comment what you think it should do ? Like a step by step explanation of what you think it actually does.

Reply:
@epicallan , 

#According  to your latter comment

- the function takes in an array _*all_* issues
-then filters out only those commented on issues
-then returns them sorted by title and a list of comments on a specific issue

#however, according to my first interpretation, 
-ideally the function was to create issues with comments and then add the to a particular repo in the names of the author i.e 
`   const fields = {
      owner: "author",
      repo: "repository",
      title: "title",
      body: "comments"
    };`